cmd/concourse/generate_key.go

```diff
@@ -74,3 +78,27 @@ func (cmd *GenerateKeyCommand) Execute(args []string) error {
 
 	return nil
 }
+
+type ExtractInternalConfigCommand struct {
+	FilePath string `short:"f"  long:"filename"  required:"true"  description:"File path where the key shall be created. When generating ssh keys, the public key will be stored in a file with the same name but with '.pub' appended."`
```

Comment:
The `description` here appears to not be updated. It talks about ssh keys but it looks like it's actually the destination you want the seccomp profile written to.

Reply:
Maybe change the `longname` to something else to, like `output-file`.